src/senfd/documents/enriched.py

```diff
@@ -140,7 +140,10 @@ class IdentifyDataStructureFigure(EnrichedFigure):
     REGEX_FIGURE_DESCRIPTION: ClassVar[str] = r".*Identify.*Data.Structure.*"
     REGEX_GRID: ClassVar[List[Tuple]] = [
         REGEX_GRID_RANGE,
-        REGEX_GRID_REQUIREMENTS,
+        REGEX_GRID_RANGE,
+        REGEX_GRID_IO,
+        REGEX_GRID_ADMIN,
+        REGEX_GRID_DISCOVERY,
```

Comment:
Same question regarding the "discovery". Will this limit the matches for non-fabrics stuff?